apps/prairielearn/src/lib/navPageTabs.ts

```diff
@@ -36,10 +36,10 @@ export function getNavPageTabs(hasEnhancedNavigation: boolean) {
           },
 
           {
-            activeSubPage: 'access',
-            urlSuffix: '/instance_admin/access',
+            activeSubPage: 'publishing',
+            urlSuffix: '/instance_admin/publishing',
             iconClasses: 'far fa-calendar-alt',
-            tabLabel: 'Access',
+            tabLabel: 'Publishing',
```

Comment:
For lack of a better place for this comment, the "Instance settings" option on the sidebar isn't highlighted when this page is active. Note that it is when the "General" tab is active.

Reply:
Fixed in 7ea2f57dca28dbeae742d7c993571ee411fc1eda.